Source/SwiftyJSON/SwiftyJSON.swift

```diff
@@ -139,9 +150,21 @@ public struct JSON {
 		if let data = jsonString.data(using: .utf8) {
 			self.init(data)
```

Comment:
```suggestion
		    self.init(data)
```

I know we should stick to the style of the author, but this indentation is most likely accidentally, as other `if`s are also indented with four spaces